src/games/gameitem.py

```diff
@@ -214,103 +214,31 @@ def tooltip(self, data):
         game = data.game
         teams = self._game_teams(game)
         observers = self._game_observers(game)
-        return self._tooltip_formatter.format(teams, observers, game.sim_mods)
+        return self._tooltip_formatter.format(teams, observers, game.sim_mods,)
```

Comment:
,

Reply:
weird, how did that slip in - i didnt even get an exception for that though oO? python being random i guess

Reply:
git add -ip